src/win32/mouse.cpp

```diff
@@ -14,6 +14,10 @@ Point GetPosition() {
 void MoveAbsolute(int x, int y) {
     // (0, 0) is the top left corner
     // (65535, 65535) is the bottom right corner
+
+    x = x * 65535 / GetDeviceCaps(GetDC(NULL), HORZRES);
```

Comment:
It looks like `GetDC` and `GetDeviceCaps` will be called every time `MoveAbsolute` is called.

Could the screen size (width and height values) be cached in the `_Context` class?
The `_Context` class is defined in `src/internal.h` and the win32 implementation is in `src/win32/context.cpp`.